repos/keg-components/src/components/typography/caption/index.js

```diff
@@ -0,0 +1 @@
+export * from './caption'
```

Comment:
I'm not sure I follow the need for doing this?
Not just in this file, but for this and all the rest of them?

Reply:
on this comment
> Some of the folder structure changes do make sense? Why move some many files into folders
I'm gonna guess you mean it does **not** make sense?

1. I think originally I was grabbing the file based on the `index.js` file so the file structure being what it is now really helps with that process (and it was somewhat consistent). 
2. Now that I'm grabbing the file a different way, the file structure wont help or hurt my process. so i can revert most of this back.
3. i think for files that has a .native counterpart. it's still helpful to put them in a folder. so I think i want to keep those, what do you think?
    - ex: `/useAnimate/useAnimate.js` && `/useAnimate/useAnimate.native.js`

Reply:
updated to only move file into folder if it has a .native counterpart